src/server/v2.0/handler/artifact.go

```diff
@@ -501,11 +501,12 @@ func (a *artifactAPI) RequireLabelInProject(ctx context.Context, projectID, labe
 	return nil
 }
 
-func option(withTag, withImmutableStatus, withLabel, withAccessory *bool) *artifact.Option {
+func option(withTag, withImmutableStatus, withLabel, withAccessory *bool, latestInRepository *bool) *artifact.Option {
```

Comment:
It is better to refactor the current option like this 
https://github.com/goharbor/harbor/blob/e92674a42ad7461e085db00893f6e16e8cc8d080/src/lib/cache/options.go